### ISSUE TITLE
Screen reader is announcing the selected item twice

### ISSUE THREAD

**Issue by dabrauns** [microsoft/fluentui#22188]
#### Actual behavior:

![image](https://user-images.githubusercontent.com/75364765/159671366-ea576513-4cb9-4e83-9b1f-f18f5f4db5d1.png)

#### Expected behavior:

NVDA should not announce the selected option twice.

**Comment by smhigley** [comment]
The basic issue is that support for the following combobox markup is not yet at 100%:
`<div role="combobox">value text</div>`.

Firefox does not expose the text content of a non-input combobox as its value, so we currently include the value text as part of the accessible name as a workaround. It's overall better for some users to get the value twice (a slightly more verbose experience) than for others to not be able to access it at all (a fully blocking experience).

When Firefox does add support for value-from-content, we can remove this. The ARIA WG and ARIA-AT project is already working to get better support for this (and support is improving, it's just a relatively new pattern).
